CharacterCreation/01_03_Races_Tundari

```diff
@@ -0,0 +1,171 @@
+also see well in daylight. 
+
+Culturally and economically, the clans on the Tunadri homeworld have an instated 
+breeding season once every four years. Cubs born outside of this season are not 
+harmed, but their parents are firmly reprimanded for the first, and drastic measures 
+may be taken if a single family has multiple children outside of the typical breeding 
+times. Rather than the byproduct of a controlling government, these mandates came 
+into effect nearly four-hundred years prior. As infant mortality rates decreased, the
+population increased far more rapidly than their ecosystem could handle. They have 
+kept their population stable for the last three centuries, and no one seems inclined 
+to change their ways. Space exploration and colonization have begun to change that, 
+but many colonists still ascribe to the old mandates.
+
+Types:                            Biological
+                                  Mammalian
+                                  Warm-blooded 
+                                  Carnivorous 
+
+Weakness to Poisons/Contagions:   [Insert Poison rules here]
+
+
+===== Language: Tuunra ===== 
+
+While different clans have their own unique dialects, they are similar enough that 
+they can be understood well by all. Tuunra features drawn-out vowels, soft consonants,
+and sounds such as clicks, chirps, trills, and a low rumbling that is nearly inaudible 
+to humans. There is another integral part of Tuunra that many species do not pick up
+on, and that is an intricate system of body language that involves both tail and ears.
+A Tundari that has lost their tail may have great difficulty getting their point across
+to others of their species.
+Tundari can communicate basic ideas and general instructions noiselessly. 
+
+===== Names ===== 
+
+First names range very much as humans, as far as origin. Often having some significance 
+with the family, first names are fairly arbitrary in much of Tundari society. Tribe and 
+clan names are held the most important, and as such, are always listed first.
+Some Tundari have been exiled, lost their clan names, or adopted their own. Others have 
+adopted names from the other cultures they now live in. 
+
+                      ===== Tuunra Name Examples =====
+                              Rra’Igri Chi’ro
+                              Sang’Aal Meari
+                              Chi’Mer Brea
+                              Chi’Krou Amri
+                              A’rhe Gaur
+
+===== Available Traits =====
+
+Clawed:
+Already equipped with built-in weapons, the large claws on your forelimbs give you an 
+advantage in hand-to-hand combat. 
+[Increased Damage for Hand-to-Hand]
+
+Soft-footed:
+An ambush predator, you are built for sneaking up on those around you. You know how to 
+handle your weight and use the knowledge passed down by generations of hunters to keep 
+your presence a secret. 
+[Stealth Rolls may be Averaged]
```

Comment:
I especially love this flavor text! It conveys a sense of fun and power while sticking very well to the mechanical benefit! Aces!